src/Microsoft.Experimental.Collections/Microsoft.Experimental.Collections.csproj

```diff
@@ -0,0 +1,24 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="..\..\tools\common.props" />
+  <PropertyGroup>
+    <Description>This package provides collections that are being considered for future versions of .NET Core.</Description>
```

Comment:
What about something like `This package contains some useful collections that are not currently included in .NET Core itself.`

Reply:
Oh, I see you are using the existing description. It's fine let's stick with that.

Reply:
I did change the description, the base is the same, just changed .NET Framework -> .NET Core.